frontend/src/routes/(authed)/player/[id]/+page.server.ts

```diff
@@ -33,6 +33,10 @@ export const load: PageServerLoad = async ({
   const lost = totalMatches - wins;
   const winrate = totalMatches > 0 ? wins / totalMatches : 0;
 
+  const msSinceLastMatch =
+    new Date(matches[0].date).getTime() - new Date().getTime();
```

Comment:
This breaks if user hasn't played a single match. We should probably handle that?

Reply:
Fixed. I forgot that players could exist before having played a match